test/factories/groups.rb

```diff
@@ -4,20 +4,25 @@
     type { :group }
     sequence(:id, 400)
 
+    after(:create) do |group|
+      group.memberships << create(:membership, :leader, group: group)
+    end
+
```

Comment:
Is there any reason to add leader to the `basic_group`? I think this should be as lightweight as possible and the leader should remain in the `group`.

Reply:
i added leader to the `basic_group` because I needed the `group.parent.leader.user` to work in order to test resort leader access.

Reply:
In this case I'd recommend to change the related factory/initialization to use the `group` instead of the `basic_group`. My problem here that you cannot create a lightweight group w/o leader if it's necessary.